specs/carousel/carousel.md

```diff
@@ -0,0 +1,282 @@
+>
+
+
+### Use Cases
+
+- Jeff is looking for an apartment and wants to see what the apartment looks like, so he clicks on the main image and a carousel appears allowing him to rotate through the images of the apartment.
+
+- Jess is checking out the latest news from her favorite website and quickly sees the main headlines of the day at the top of the webpage as they rotate headline by headline across the page.
+
+### Features
+
+- **Next and Previous Controls:** Simple buttons [aka flippers] that allow for control of displaying the next and previous slides.
+
+- **Slide Picker:** A control or group of controls that allows the specific choice of slide to display.
+
+- **Autoplay:** Allows the carousel to begin rotating through slides automatically after the carousel loads. Autoplay can be toggled on/off with a button, and automatically stops when any element in the carousel receives keyboard focus or is hovered over.
+
+- **Looping:** Allows the user to navigate from the first slide to the last if the previous flipper is activated. If on the last slide the user can navigate from the last slide to the first when the next flipper is activated.
+
+
+### Risks and Challenges
+
+*Notable risks or challenges associated with implementing the component. Would we need to make any breaking changes in order to achieve this component's goals?*
+
+### Prior Art/Examples
+- [FAST-DNA (React)](https://explore.fast.design/components/carousel)
+- [W3C](https://w3c.github.io/aria-practices/examples/carousel/carousel-1.html)
+
+---
+
+## Design
+
+### API
+
+*Component name:*
+- `fast-carousel`
+
+*Attributes & Properties:*
+- `autoplay` - boolean value to toggle if the carousel plays on load or not (defaults to false)
+- `autoplay-interval` - a number representing the autoplay interval in milliseconds
+- `loop` - boolean value to toggle if the carousel loops slides at the beginning or end (defaults to true)
+- `aria-labelledby` - optional based on implementation**
+- `aria-label` - optional, based on implementation**
+- `paused` - whether the rotation is paused or not
+
+** See the [W3C Specification](https://w3c.github.io/aria-practices/#wai-aria-roles-states-and-properties-4) for requirements and details.
+
+*CSS custom properties:*
+- ???
+
+*Slots:*
+- `default` - slot for slides content // TODO: IS this the right way?
+- `play-toggle` - the control to toggle if the slides rotate or not
+- `next-flipper` - the control used to rotate to the next slide
```

Comment:
I kind of think "flipper" isn't the right term here. That is the name of the control *we* use for this, but it's really just a button. Thoughts on changing this to somthing like `next-button` / `previous-button`?